### ISSUE TITLE
Allow saving of a project

### ISSUE THREAD

**Issue by hansdemol** [debkbanerji/lego-art-remix#113]
Hi Deb,

I often like to 'come back to' / improve a conversion I have worked on before.
Therefore, I would really appreciate if it would be possible to 'save' a mosaic project (with the picture, the color palette, the adjustments,...) and to be open it again a couple of days later. (I rarely consider a conversion ready/perfect/done immediately.)

Also, when building in real life, I sometimes make adjustments in the colors chosen/used and I would like to reflect that in the .png and .pdf (especially when I want somebody else to rebuild the mosaic afterwards). And then it would be nice to reopen the project as well...

Could this feature be implemented?

Thanks for your thoughts,

Hans

**Comment by loopanyway** [comment]
Love the Idea!

**Comment by Rukario** [comment]
That can be partially implemented as cookie that holds up to 4096 bytes. All adjustments can be saved for next page visit, but pictures tend to hit past 4kb so in that case: filename remembered and ask to reupload the picture.

**Comment by Sizzlin1986** [comment]
I agree with the idea behind adding a saving feature. Even if it was as simple as a settings printout on the instructions. That way if I go back to tweak the same image, I could reupload the original image, input the original settings, and then change from there.

**Comment by jheyday** [comment]
I forked and added the ability to save and a few other nice features like undo/redo, floodfill, custom section sizes
